src/frontend/sharedComponents/AddButton.tsx

```diff
@@ -27,16 +26,9 @@ const AddButtonNoMemo = ({
 }: AddButtonProps) => (
   <View
     testID={testID}
-    style={[styles.container, {bottom: isLoading ? 75 : 25}, style]}>
+    style={[styles.container, {bottom: isLoading ? 50 : 25}, style]}>
     <TouchableOpacity disabled={disabled || isLoading} onPress={onPress}>
-      {!isLoading ? (
-        <Image
-          source={require('../images/add-button.png')}
-          style={styles.button}
-        />
-      ) : (
-        <Loading size={15} />
-      )}
+      {isLoading ? <UIActivityIndicator size={50} /> : <AddButtonSVG />}
```

Comment:
I also updated this loader. This was a legacy thing before we were using UiActivityIndicator. Updating it to be consistent with the rest of the app